scripts/docgen/api/findUnusedExports.ts

```diff
@@ -45,12 +44,72 @@ export const findUnusedExports = () => {
     return unusedExports
 }
 
+type UnusedFileExportsContext = {
+    sourceFile: SourceFile
+    apiExports: ApiExports[]
+    file: string
+}
+
+const findUnusedExportsInFile = (context: UnusedFileExportsContext) => {
+    const exportedDeclarations = context.sourceFile.getExportedDeclarations()
+    return findUnusedDeclarations(exportedDeclarations, context)
+}
+
+const findUnusedDeclarations = (
```

Comment:
Lucky we don't have the function length linting rule on in the scripts directory LUL

Also this is perfect and a huge improvement thank you <3